### PR TITLE
Updated 01.abstract.md

### PR DESCRIPTION
**Pull request by slochower**
I've added a sentence that mentions the Manubot article under review.

### CODE REVIEW

**Comment by agitter** [comment]
It would be untraditional to have a citation in abstract, but should we add one to meta-review anyway to help reviewers find the paper?

File: content/01.abstract.md [hunk 1]
```diff
@@ -5,3 +5,5 @@ Manubot applies the git-based software workflow to scholarly writing, enabling e
 
 This manuscript is a Manubot demo, intended to give users a playground to practice using Manubot.
 Everyone is encouraged to try writing with Manubot by editing this manuscript.
+
+Manubot is described in the paper titled "Open collaborative writing with Manubot".
```
Comment (agitter) [inline content/01.abstract.md]: Please add a URL citation to this paper.

**Review by agitter — CHANGES_REQUESTED** [review]
Thanks for the contribution.  I have one suggestion below.

File: content/01.abstract.md (hunk 1 above)
Comment (slochower) [reply content/01.abstract.md]: That's a good idea!

Comment (slochower) [reply content/01.abstract.md]: That's a good idea!

Comment (slochower) [reply content/01.abstract.md]: I've added the citation and the manuscript is ready for re-review.

**Review by agitter — APPROVED** [review]

**Pull request merged**